src/deprecation.js

```diff
@@ -5,6 +5,26 @@ var core = require('./core'),
     extras = require('./extras'),
     filters = require('./filters');
 
+// provide method to give a stack track for warnings
+// useful for tracking-down where deprecated methods/properties/classes
+// are being used within the code
+function warn(msg) {
+    var stack = new Error().stack; 
+
+    // chop off the stack trace which includes pixi.js internal calls
+    stack = stack.split('\n').splice(3).join('\n');
```

Comment:
I checked on Edge and it needs a "4" splice parameter to work. The stack property being non standard, I think that @englercj idea make sense here.